bundles/org.openhab.binding.froniuswattpilot/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="froniuswattpilot"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Thing Types -->
+	<thing-type id="wattpilot">
+		<label>Fronius Wattpilot</label>
+		<description>Fronius Wattpilot wallbox</description>
+
```

Comment:
Could you add an equipment tag. 
For reference see: https://next.openhab.org/docs/developer/bindings/thing-xml.html#thing-semantic-equipment-tags